Terminal.Gui/View/Layout/ViewLayout.cs

```diff
@@ -437,7 +437,13 @@ Dim VerifyIsInitialized (Dim dim)
 	/// during set operations and in <see cref="LayoutSubviews"/>.If invalid settings are discovered exceptions will be thrown indicating the error.
 	/// This will impose a performance penalty and thus should only be used for debugging. 
 	/// </remarks>
-	public bool ValidatePosDim { get; set; }
+	public bool ValidatePosDim {
+		get => _validatePosDim;
+		set {
+			_validatePosDim = value;
+			ResizeView (AutoSize);
```

Comment:
I don't think turning on/off a diagnostic should have side-effects like calling ResizeView. Why do you think this is necessary here?

Reply:
If you comment `ResizeView (AutoSize);` the `AutoSize_False_SetWidthHeight_With_Dim_Fill_And_Dim_Absolute_With_Initialization` unit test will fail because `label.AutoSize = true;` and set `label.ValidatePosDim = false;` will resize the label for the text size.